submissions/nwokejeme-immaculeta/dom/signup.js

```diff
@@ -0,0 +1,66 @@
+let submitForm = document.getElementById('validateForm'),
+    firstName = document.getElementById('firstName'),
+    lastName = document.getElementById('lastName'),
+    otherName = document.getElementById('otherName'),
+    phoneNumber = document.getElementById('phoneNumber'),
+    email = document.getElementById('email');
+    password = document.getElementById('password');
+    confirmPassword = document.getElementById('confirmPassword');
+
+    let validate = (field, event, regEx, result, output) => {
+        if (field.value === "") {
+            field.nextElementSibling.innerHTML = result;
+            event.preventDefault();
+            return false;
+        } else
+        if (!field.value.match(regEx)) {
+            field.nextElementSibling.innerHTML = output;
+            event.preventDefault();
+            return false;
+        } else {
+            field.nextElementSibling.innerHTML = "";
+            return true;
+        }
+    }
+
+    let passwordValidation = (password1, password2, event) => {
+        const passwordRegex = /(?=.*\d)(?=.*[a-z])(?=.*[a-zA-Z]).{6,}$/;
+    
+        if ((password1.value && password2.value) === "") {
+            password1.nextElementSibling.innerHTML = "Please enter your password";
+            password2.nextElementSibling.innerHTML = "Please confirm your password";
+            event.preventDefault();
+            return false;
+        } else if ((!password1.value.match(passwordRegex)) && (!password2.value.match(passwordRegex))) {
```

Comment:
This check is already handled in your `validate` function. You are repeating yourself here